lib/components/Modal/Bottom/modal-body.tsx

```diff
@@ -0,0 +1,19 @@
+import clsx from "clsx";
+import { Text } from "../../Typography";
+
+interface ModalBodyProps {
+    className?: string;
+}
+
+export const ModalBody = ({
+    children,
+    className,
+}: React.PropsWithChildren<ModalBodyProps>) => (
+    <Text
+        size="md"
+        as="div"
```

Comment:
I suggest using `div` instead of `Text` here and simply let developers pass any elements:  strings, `<Text` components with the `size`, font-weight, etc., - which they like, as `children` :)

Reply:
Actually, I'm not sure about this. I thought content inside ModalBody should have style according to Figma by default.

Reply:
Same with ModalTitle